target/bin/sedfile

```diff
@@ -9,6 +9,17 @@
 # Is a file change expected? --> use 'sedfile --strict -i'
 # Is a file change only on the first container run expected? --> use 'sedfile -i'
 
+if [[ -e /usr/local/bin/helpers/log.sh ]]
+then
+  # shellcheck source=../scripts/helpers/log.sh
+  source /usr/local/bin/helpers/log.sh
+else
+  # when running BATS (unit tests), this file is not located
+  # inside a container; as a consequence, we need to source
+  # from a different location
+  source target/scripts/helpers/log.sh
```

Comment:
Why is this exactly? It's built in the Dockerfile, are we calling it in a test somewhere outside of the container for some reason?

Reply:
> are we calling it in a test somewhere outside of the container for some reason?

Yes, the `sedfile.bats` test executes the script directly, not in a DMS container. I think this makes sense though and it's fine - but it also requires this "workaround".

Reply:
> I think this makes sense though and it's fine - but it also requires this "workaround".

I don't see [anything in the test](https://github.com/docker-mailserver/docker-mailserver/blob/master/test/sedfile.bats) that requires it to run a copy in the host systems `/tmp`?

I would think it more appropriate to be testing the `sedfile` within the image we've built as the test suite already assumes a docker image for pretty much everything else?

---

I'm not going to push for that, it just seems a little out of place is all.

Reply:
I concur, maybe @casperklein can update this in the future if he agrees as well :)

Reply:
We all agree, this solution is ugly. I did not run the `sedfile` tests in a container, just because at this point there was no reason to do + I am an all-time fan of avoiding unnecessary overhead 😎
I will adjust the bats test in a future PR to run inside a container.